DBSCE/KeyGeneration.md

```diff
@@ -0,0 +1,91 @@
+### Local key helper on Windows
+
+On Windows, a Local key helper is a COM class. A COM interface that the local key helper implements is TBD (working name for this document is ILocalKeyHelper)
```

Comment:
is a COM class per domain?  Or the same class registered for multiple domains?  Could I use *.contoso.com in the example below on line 33?